src/Samples/Sarif.Sdk.Sample/Program.cs

```diff
@@ -226,7 +239,7 @@ static int CreateSarifLogFile(CreateOptions options)
                         var result = new Result()
                         {
                             RuleId = rule.Id,
-                            AnalysisTarget = new FileLocation { Uri = new Uri(@"file://d:/src/module/foo.dll") }, // This is the file that was analyzed
```

Comment:
Not MS compliant

Reply:
Hooray! My pet peeve addressed.

---
In reply to: [210981653](https://github.com/Microsoft/sarif-sdk/pull/1024#discussion_r210981653) [](ancestors = 210981653)